examples/hashing/src/main.sw

```diff
@@ -35,30 +34,50 @@ fn main() {
     let sha_hashed_u32 = sha256(~u32::max());
     let sha_hashed_u64 = sha256(~u64::max());
 
+
```

Comment:
Additional new lines? This happens in multiple files

Reply:
Ah this is from the previous formatting I did, I'll go in and fix that rq

Reply:
As it turns out this is actually a side effect of the previous bug fix for preceding newlines on comments. I'll reopen the issue.

Reply:
#2691

Reply:
I added a bandaid fix for now, and will continue looking into it further